gnowsys-ndf/gnowsys_ndf/ndf/management/commands/event_update_status.py

```diff
@@ -0,0 +1,71 @@
+from gnowsys_ndf.ndf.models import *
```

Comment:
We can remove this file now, as we will not be applying cron job for updating event status.